src/_common/fireside/post/card/AppPostCardWithText.vue

```diff
@@ -0,0 +1,229 @@
+<script lang="ts">
```

Comment:
@hworld do we need this? it's currently unused, and `AppPostCard.vue` has some other changes recently.